### ISSUE TITLE
Get the following error when stopping an acquisition.  Calling 'shutdown( 2 )' failed with error 'Transport endpoint is not connected'

### ISSUE THREAD

**Issue by BaileyHelfer** [genicam/harvesters#341]
**Describe the Bug**
On Linux Ubuntu 20.04 using Matrix Vision 2.41.0 CTI file. I get the following error "Calling 'shutdown( 2 )' failed with error 'Transport endpoint is not connected'" anytime I try and call image Image Acquirer's stop function. I have tried using Harvesters 1.3.3, 1.3.6 and 1.3.7 but I get the error with all of them.

**To Reproduce**

```
h = Harvester()
gentl_file = "/opt/mvIMPACT_Acquire/lib/x86_64/mvGenTLProducer.cti"
h.add_file(gentl_file)
h.update()

ia = h.create(0)

ia.start()
with ia.fetch() as fetch:
    print(fetch)
ia.stop()
ia.destroy()
   
```

When running this same code on windows with the same CTI version we do not get this error. Possibly an error only when using Linux Ubuntu?

**Comment by kazunarikudo** [comment]
@BaileyHelfer Hi, thanks for the report.

> Possibly an error only when using Linux Ubuntu?

I am not the right person to answer this but it should be reasonable to think they are not identical from binary compatibility point of view and the drivers should be different, of course.

Concerning the issue itself, I would appreciate it if you could try to run the same script on another computer so that you can check the reproducibility.

Thanks, Kazunari.

**Comment by BaileyHelfer** [comment]
I know it has been awhile but figured I'd comment back here on my findings in case anyone else came across this message. I was able to reproduce this on other Linux machines so I figured it had to be with either the OS or the CTI file that we were installing. We got this message when installing MV_Acquire and using their CTI file. When using another vendors software like Baumers who also does not have a vendor lock, the message was no longer there when calling ia.stop(). So in short, the issue must be with something during the install of the MV_Acquire software.
